com.github.sdbg.debug.ui/src/com/github/sdbg/debug/DebugLaunch.java

```diff
@@ -0,0 +1,37 @@
+package com.github.sdbg.debug;
+
+import com.github.sdbg.debug.ui.internal.DartUtil;
+import com.github.sdbg.debug.ui.internal.chrome.ChromeLaunchShortcut;
+import com.google.gdt.eclipse.core.sdbg.IDebugLaunch;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+
+public class DebugLaunch implements IDebugLaunch {
+
+  @Override
+  public boolean hasChromeLauncher() {
+    return true;
+  }
+
+  @Override
+  public void launchChrome(IProject project, String mode, String url) {
+    if (project == null || mode == null || url == null) {
+      DartUtil.logError("DebugLaunch.launchChrome has a null. project=" + project + " mode=" + mode
+          + " url=" + url);
+      return;
+    }
+
+    IJavaProject javaProject = JavaCore.create(project);
+    if (javaProject == null) {
+      DartUtil.logError("DebugLaunch.launchChrome can't figure out javaProject=" + javaProject);
+    }
+
+    IResource resource = javaProject.getResource();
+
+    new ChromeLaunchShortcut().launch(resource, mode, url);
```

Comment:
I didn't want to extract the logic at this point. I'm looking for the minimum amount of changes I can get away with.